src/NativeShell/src/NativeShell.java

```diff
@@ -251,4 +254,56 @@ private boolean hideMediaSession() {
         callbackContext.sendPluginResult(pluginResult);
         return true;
     }
+
+    private boolean download(String url) {
+        String title;
+        String filename;
+        try {
+            JSONObject options = args.getJSONObject(0);
+            title = options.getString("title");
+            filename = options.getString("filename");
+        } catch (Exception e) {
+            callbackContext.error("error: download: " + e.getMessage());
+            return false;
+        }
+        Context context = cordova.getContext();
+        Uri uri = Uri.parse(url);
+
+        DownloadManager.Request request = new DownloadManager.Request(uri)
+                .setTitle(title)
+                .setDescription("Downloading...")
+                .setDestinationInExternalPublicDir(Environment.DIRECTORY_DOWNLOADS, filename)
+                .setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE_NOTIFY_COMPLETED);
+
+        cordova.getActivity().runOnUiThread(() -> {
+            // TODO translate these strings
```

Comment:
When is this going to be done? IMHO, I wouldn't release any Android update without the strings being translatable. Can't we simply reuse the strings from Web?

Reply:
I don't know any of that... Depends on the architecture, I just copied the Todo from above

Reply:
There's no example how to use translated strings in this architecture, so I leave it to you guys

Reply:
I would better have a non-translated feature that no feature at all, though.